### PR TITLE
Added new enemy type "Gremlin"

### PR DESCRIPTION
**Pull request by HubiKazak**
Nothing much, might add more enemy types if I don't forget, It's basically first open-source I gave a reasonable contribution to, so I guess "Hello World"

### CODE REVIEW

**Comment by tagniam** [comment]
Hey, thanks for contributing! Looks like there might be some bugs though if you check the build.

**Comment by HubiKazak** [comment]
Added the missing comma, should be working now ^^

**Comment by HubiKazak** [comment]
Greens all across the board. Thanks for chance to contribute

File: Turn/include/Enemy.h [hunk 1]
```diff
@@ -26,7 +26,8 @@ enum EnemyType
     etSlimeball,
     etRatKing,
     etZabra,
-    etNumEnemyTypes   
+    etNumEnemyTypes,   
+	etGremlin
```
Comment (tagniam) [inline Turn/include/Enemy.h]: Indentation

File: Turn/include/EnemyTypes/Gremlin.h [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+#include "../Enemy.h"
+
+
+class Gremlin : public Enemy 
+{
+public:
+	Gremlin();
+	
+    EnemyType GetType() override;
```
Comment (tagniam) [inline Turn/include/EnemyTypes/Gremlin.h]: Indentation

File: Turn/src/Game.cpp [hunk 3]
```diff
@@ -242,6 +242,10 @@ void Game::SetEnemy(){
 		case etZabra:
 			_Enemy = new Zabra;
 			break;
+		case etGremlin:
+		//Enemy is a Gremlin
```
Comment (tagniam) [inline Turn/src/Game.cpp]: Indentation

File: Turn/include/Enemy.h (hunk 1 above)
Comment (tagniam) [inline Turn/include/Enemy.h]: Indentation

File: Turn/src/Game.cpp (hunk 3 above)
Comment (tagniam) [inline Turn/src/Game.cpp]: Indentation

File: Turn/include/Enemy.h (hunk 1 above)
Comment (HubiKazak) [reply Turn/include/Enemy.h]: This one looks really weird, it is spaced out properly on my local version, I'll look further into it

Comment (HubiKazak) [reply Turn/include/Enemy.h]: Resolved in next commit

**Comment by tagniam** [comment]
Thanks @HubiKazak! Happy Hacktober :tada:

**Pull request merged**